airflow/www/static/css/main.css

```
@@ -105,7 +119,17 @@ div.container {
 @media (prefers-reduced-motion: no-preference) {
   .navbar-brand:hover .brand-logo-pinwheel {
     transform-origin: 17.66px 17.66px;
-    animation: pinSpin 1.5s linear;
+    animation: pinSpin 1.5s linear infinite;
```

```suggestion
    animation: pinSpin 1.5s linear;
```
We wish to not have an infinite spin. There was an issue https://github.com/apache/airflow/issues/34019 logged previously about this and hence we removed the infinite here https://github.com/apache/airflow/pull/34020

---

I mention in the second paragraph of the description why I think removing infinite spin was not the right action to take away from issue #34019

---

Okay, thanks, sorry missed reading that 😐